client/src/components/Ticketing/ticketingmanager/Refund/RefundOrderTicket.tsx

```diff
@@ -1,42 +1,174 @@
+                    </svg>
+                  </div>
+                </button>
+              </div>
+            </label>
+          </div>
+          <div className='w-full min-w-min'>
+            <div className='grid grid-cols-5 gap-2 bg-gray-200 h-18 rounded-lg shadow-md px-2 mb-2 font-bold'>
+              <div className='row-start-1 justify-self-start py-2 col-span-1'>
+                  Name
+              </div>
+              <div className='row-start-1 justify-self-start py-2 col-span-1'>
+                <div className="border-l border-l-gray-500/50 pl-2">
+                  Date
+                </div>
+              </div>
+              <div className='row-start-1 justify-self-start py-2 col-span-1'>
+                <div className="border-l border-l-gray-500/50 pl-2">
+                  Showing(s)
+                </div>
+              </div>
+              <div className='row-start-1 justify-self-start py-2 col-span-1'>
+                <div className="border-l border-l-gray-500/50 pl-2">
+                  Total
+                </div>
+              </div>
+              <div className='row-start-1 justify-self-center py-2 col-span-1'></div>
             </div>
-            <button className='bg-red-700
-              hover:bg-red-500 text-white mt-5 py-2 px-4 rounded'>
-              Refund
-            </button>
+            {mappedInstances.length === 0 ? (
+              <div className="text-center text-gray-600">No current results</div>
+            ) : (
+              mappedInstances.map((instance, index) => (
+                <div key={index} className='grid grid-cols-5 gap-2 bg-gray-200 rounded-lg shadow-md px-2 mb-2 hover:bg-gray-300'>
+                  <div className='row-start-1 justify-self-start pl-2 py-2 col-span-1'>
+                    {instance.name}
+                  </div>
+                  <div className='row-start-1 justify-self-start pl-2 py-2 col-span-1'>
+                    {instance.date}
+                  </div>
+                  <div className='row-start-1 justify-self-start pl-2 py-2 col-span-1'>
+                    {Array.isArray(instance.showings) ? (
+                      instance.showings.map((showing, showingIndex) => (
+                        <div key={showingIndex}>{showing}</div>
+                      ))
+                    ) : (
+                      <div>{instance.showings}</div>
+                    )}
+                  </div>
+                  <div className='row-start-1 justify-self-start pl-2 py-2 col-span-1'>
+                    {instance.price}
+                  </div>
+                  <div className='row-start-1 justify-self-start py-2 col-span-1'>
+                    <button
+                      onClick={async () => await onSubmit(instance.orderID)}
+                      type={'submit'}
+                      className='bg-red-700 hover:bg-red-800 text-white py-2 px-4 rounded'
+                    >
+                      Refund
```

Comment:
When refunding the John Doe ticket, is it suppose to remove it from the table? When I click Refund, it says the refund was unsuccessful. However, I saw in the picture you provided in the description that it was removed for you? Just wanted some clarifications on that.

Reply:
Currently not implemented, I'll see if this can be done front-end or requires an update/refresh to the db after refunding

Reply:
That is implemented on branch 95.